### ISSUE TITLE
Add method to docket objects to reprocess themselves, then run it on entire corpus.

### ISSUE THREAD

**Issue by mlissner** [freelawproject/courtlistener#800]
Basically, I want to do:

    d = Docket.objects.get(pk=1)
    d.reprocess_recap_content()

That'd make life easier to have that around.

**Comment by mlissner** [comment]
I need to reparse the entire corpus as part of #755, so making that part of this issue too.

**Comment by mlissner** [comment]
In ee7d0e5f34da76d851564b3b76be6c4b28b08a77, we now have a way of reparsing the HTML files, and that's fixing up some of the damage in #799. 

Next step is to make it work on the XML files. To do that, I'm going to make a new report parser for Juriscraper. That'll allow us to use the same processing/merging code for the API, Internet Archive XML, and reparsing. Too much code duplication in there now.

I embraced the code duplication at first because there was enough time between the implementations to allow the duplication to be better than the original, but now that we've got that (and refined it as the RECAP API), we can ditch the original and replace it with v2.
